index.html

```diff
@@ -14,30 +14,30 @@
     <section id="destinations" class="section-destinations">
         <div class="destinations-wrapper">
             <div class="destinations-textbox">
-                <h1>How to choose destination?</h1>
+                <h1>HOW TO CHOOSE DESTINATION?</h1>
```

Comment:
This could be done in css with text-transform: uppercase;

Reply:
good to know! will keep in mind